src/OpenReader.m

```diff
@@ -1085,4 +1083,14 @@ - (void)createFolders:(NSMutableArray *)params
     }
 }
 
+
+/**
+ Get the current timestamp
+ 
+ @return current timestamp as a string
+ */
++ (NSString *)currentTimestamp {
```

Comment:
I am happy to consider other advice like changing to instance method or if there is some document that says it is ok to use read only properties with class methods in the case of a singleton etc 👍